e2e.mk

```diff
@@ -0,0 +1,16 @@
+KINDCLUSTER=kind
+
+build-local-readiness: manifests package-vendir-sync
+	COMPONENTS=readiness/controller.readiness-controller-manager.readiness make docker-build-all
+	docker tag projects.registry.vmware.com/tanzu_framework/readiness-controller-manager:latest readiness-controller-manager:latest 
+
+deploy-local-readiness: build-local-readiness
+	kind create cluster --name ${KINDCLUSTER}
```

Comment:
Is this required to run every time ??

Reply:
not really. We can make it optional if the cluster is pre-created.